### ISSUE TITLE
Socket InvalidStateError: The connection has not been established yet

### ISSUE THREAD

**Issue by etler** [avocode/avocode#1122]
IMPORTANT: Please try to describe how did it happen.

Did nothing, just happened randomly while the program was idle in the background.
-----
App v. `2.10.1`
Source UID: 57855
```
Error: InvalidStateError: The connection has not been established yet
    at SockJS.send (/home/tim/Avocode/resources/app.asar/node_modules/sockjs-client/lib/main.js:158:11)
    at e.value (file:///home/tim/Avocode/resources/app.asar/dist/avocode.min.js:5:23588)
    at e.value (file:///home/tim/Avocode/resources/app.asar/dist/avocode.min.js:5:23092)
    at file:///home/tim/Avocode/resources/app.asar/dist/avocode.min.js:5:22957
    at /home/tim/Avocode/resources/app.asar/node_modules/immutable/dist/immutable.js:3907:58
    at /home/tim/Avocode/resources/app.asar/node_modules/immutable/dist/immutable.js:1379:16
    at ArrayMapNode.iterate.HashCollisionNode.iterate [as iterate] (/home/tim/Avocode/resources/app.asar/node_modules/immutable/dist/immutable.js:1728:11)
    at Map.__iterate (/home/tim/Avocode/resources/app.asar/node_modules/immutable/dist/immutable.js:1377:32)
    at Set.__iterate (/home/tim/Avocode/resources/app.asar/node_modules/immutable/dist/immutable.js:3907:24)
    at Set
```

**Comment by jakubzitny** [comment]
Hi @etler, thanks for reporting this. It looks like it's happening to several users already, it's a bug and we'll have to fix it for the next release. Is there any chance you have more information about this? When did it happen, what were you trying to do?

Thanks a lot and stay tuned for updates 😉

**Comment by etler** [comment]
This was after bringing my laptop out of sleep mode, maybe half an hour
later

**Comment by connor-bean** [comment]
We just released 2.11 so I'm closing this out. Please let us know if this problem persists.

Thanks for your patience on this.

Connor, Avocode Support
